azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/manager/AudioSessionManager.kt

```diff
@@ -287,12 +294,22 @@ internal class AudioSessionManager(
                 audioManager.startBluetoothSco()
                 audioManager.isBluetoothScoOn = true
                 audioManager.isSpeakerphoneOn = false
+                val selection = AudioDeviceSelectionStatus.BLUETOOTH_SCO_SELECTED.toString()
+                notifyEventHandlerOnSelectionChange(selection)
```

Comment:
We should consider mapping to a fixed field, i.e. a enum.

3rd party Dev's won't know what to do with a string and can't trust that it won't change or have new values added.